composer.json

```diff
@@ -21,17 +21,17 @@
   ],
   "require": {
     "php": ">=7.2",
-    "nette/forms": "^3.0.0"
+    "nette/forms": "~3.0.0"
```

Comment:
Proc tu nejsou nette/forms 3.1?

Reply:
protože podle PHPStan a testů není kompatibilní

Reply:
Tak to zvedneme pro verzi 3.1.0 prosim.

Reply:
Jen nevím, jak opravit kompatibilitu..

Reply:
![obrazek](https://user-images.githubusercontent.com/6066243/146844506-0574831a-b983-40b0-a665-aae77445c943.png)